core/src/main/scala/org/apache/spark/rpc/netty/Outbox.scala

```diff
@@ -74,6 +74,14 @@ private[netty] case class RpcOutboxMessage(
     }
   }
 
+  def onCancel(): Unit = {
```

Comment:
As the implement of `onTimeout` and `onCancel` almost the same, how about combining them as onTerminate